### ISSUE TITLE
Screen reader does not announce errors when reading line by line.

### ISSUE THREAD

**Issue by aefren** [microsoft/vscode#127945]
Issue Type: <b>Performance Issue</b>

For example writing code in Python:

print(var)

And I navigates line by line with up or down key arrows, the screen reader just reads the code.

But when I go to the toolbar buttons pressing F5 I found:
1 Warnings
Warning generated by Pylance  var   is not defined at line 1 and character 7.

Detecting an error navigating line by line or word by word is an essential need for productivity in programmers, the same needs the people who uses a screen reader have.

It would be great if screen reader can read lines and word attributes as underlined or can detect if a word is marked with some colour as occur with the IDE eclipse or as Microsoft word Spelling errors  works.

VS Code version: Code 1.57.1 (507ce72a4466fbb27b715c3722558bb15afa9f48, 2021-06-17T13:28:07.755Z)
OS version: Windows_NT x64 10.0.19043
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-2500 CPU @ 3.30GHz (4 x 3300)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: unavailable_off<br>opengl: enabled_on<br>rasterization: unavailable_off<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: unavailable_off|
|Load (avg)|undefined|
|Memory (System)|7.92GB (3.67GB free)|
|Process Argv|--crash-reporter-id a872c7cd-9a7a-4263-b751-d751b55c52f7|
|Screen Reader|yes|
|VM|67%|
</details><details>
<summary>Process Info</summary>

```
CPU %	Mem MB	   PID	Process
    0	   105	  6252	code main
    0	    79	  1908	   shared-process
    0	    57	 13040	     ptyHost
    0	   108	  5092	   window (Getting Started - Visual Studio Code)
    0	    78	  3156	     extensionHost
    0	    84	  6560	   window (Issue Reporter)
    0	    22	  8728	   crashpad-handler
    0	    93	  9360	   gpu-process
    0	    38	  9400	   utility
    0	   162	 11212	   window (● probando.py - Visual Studio Code)
    0	    91	 13340	     extensionHost
    0	    67	 10412	       "C:\Users\Alfred\AppData\Local\Programs\Microsoft VS Code\Code.exe" "c:\Users\Alfred\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\json-language-features\server\dist\node\jsonServerMain" --node-ipc --clientProcessId=13340
    0	    90	 13476	       electron_node bundle.js 
    0	   133	 13284	   window (Python - Get Started - Visual Studio Code)
    0	    96	  4268	     extensionHost
    0	    62	  5464	       "C:\Users\Alfred\AppData\Local\Programs\Microsoft VS Code\Code.exe" "c:\Users\Alfred\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\json-language-features\server\dist\node\jsonServerMain" --node-ipc --clientProcessId=4268
    0	     7	  9072	       python c:\Users\Alfred\.vscode\extensions\ms-python.python-2021.6.944021595\pythonFiles\lib\python\debugpy\adapter
    0	     5	 15004	         console-window-host (Windows internal process)
    0	    64	 14164	       electron_node bundle.js 
    0	    74	 13976	   window (undefined)
```

</details>
<details>
<summary>Workspace Info</summary>

```
;
```

</details>
<details><summary>Extensions (3)</summary>

Extension|Author (truncated)|Version
---|---|---
python|ms-|2021.6.944021595
vscode-pylance|ms-|2021.6.3
jupyter|ms-|2021.6.999662501

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
pythonvspyt678:30270856
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551cf:30311713
vspre833:30321513
pythonptprofiler:30281270
vshan820:30294714
pythondataviewer:30285071
vscus158:30321503
pythonvsuse255cf:30323309
vscorehov:30309549
vscod805cf:30301675
pythonvspyt200:30331937
vscextlangct:30322307
binariesv615:30325510
vsccppwt:30329788
aatestflightcf:30331712
bridge0702:30333155

```

</details>

**Comment by isidorn** [comment]
@aefren thanks for filling this and sorry for the slow response, I was on vacation.
This is the same problem as https://github.com/microsoft/vscode/issues/90842

Until we improve our editor decoration accessibility I suggest that you either use the status bar for the problem detection (F1 > focus status bar) or the Problems panel `Ctrl + Shift + M`.

**Comment by isidorn** [comment]
Quick update: we are looking into doing quick announcements on the vscode side when a decoration is encountered (like a breakpoint). More details in this issue https://github.com/microsoft/vscode/issues/130565#issuecomment-900246306
Feedback welcome.

fyi @hediet
